app/src/main/res/layout/activity_main.xml

```diff
@@ -44,6 +44,22 @@
             android:layout_height="0dp"
             android:layout_weight="1" />
 
+        <FrameLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center">
+
+        <me.zhanghai.android.materialprogressbar.MaterialProgressBar
```

Comment:
Shall we use android provided ProgressBar

Reply:
Saw your changes, so we should add a drawable to the progress bar to fix horizontal to circle 👍